src/src/test/resources/application-test-properties/provider-gateway.properties

```diff
@@ -3,11 +3,10 @@ namespace.serialize=http://test.x-road.fi/producer
 namespace.prefix.serialize=ts1
 namespace.deserialize=http://test.x-road.fi/producer
 wrappers=false
-
 keyLength=128
-publicKeyFile=@project.basedir@/target/test-classes/application-test-keys/providertruststore.jks
+publicKeyFile=@projectDir@/build/resources/test/application-test-keys/providertruststore.jks
 publicKeyFilePassword=providerts
-privateKeyFile=@project.basedir@/target/test-classes/application-test-keys/providerkeystore.jks
+privateKeyFile=@projectDir@/build/resources/test/application-test-keys/providerkeystore.jks
```

Comment:
Could we not use `${project.projectDir}` here as we do in other properties files?